indico/modules/events/abstracts/templates/display/submit_abstract.html

```diff
@@ -0,0 +1,28 @@
+{% from 'events/display/indico/_common.html' import render_users %}
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
+
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title -%}
+    {% trans %}Submit abstract{% endtrans %}
+{%- endblock %}
+
+{%- block content %}
+    {% if cfa.is_open %}
```

Comment:
Also, checking is_open is not enough - people can be authorized to submit abstracts outside that period if this check returns True: `abstracts_settings.acls.contains_user(event, 'authorized_submitters', user)`